product-pricing-service/src/main/java/com/hedvig/contract/shared/switchers/SwitcherEmailServiceImpl.kt

```
@@ -88,6 +88,9 @@ class SwitcherEmailServiceImpl(
         content
       }
       null -> throw IllegalStateException("Cannot find line of business ${email.lineOfBusiness}")
+      SwitcherLineOfBusiness.DANISH_HOME_CONTENT -> {
+        TODO("TODO Implement email for denmark")
```

I have a faint memory that these "TODO" doesn't behave too nicely in production? Maybe throw an exception instead?

---

TODO should throw an exception this is the implementation:
```
public inline fun TODO(reason: String): Nothing = throw NotImplementedError("An operation is not implemented: $reason")
```

---

Ok nice